docs/general/administration/troubleshooting/memory-dumping.md

```diff
@@ -0,0 +1,107 @@
+---
+uid: server-memorydumps
+title: Server Memory Dumps
+---
+
+# Memory Dumping of the Jellyfin Server
+
+To troubleshoot issues with Jellyfin server related to memory allocation, it is necessary to create a memory dump in a format that developers can analyze.
+
+JetBrains dotMemory will be used for dumping memory in this guide.
+
+## Linux Barebones
+
+1. Install the latest dotMemory command line tools. To do this, download the NuGet package and extract it to a folder named `dotMemoryClt`, then set the permissions to make it executable:
+
+   ```sh
+   wget -O dotMemoryclt.zip https://www.nuget.org/api/v2/package/JetBrains.dotMemory.Console.linux-x64/2024.2.2
+   unzip dotMemoryclt.zip -d ./dotMemoryclt
+   sudo chmod +x -R dotMemoryclt/*
+   ```
+
+2. Next, determine the process ID (PID) of the Jellyfin server. You will need the `ps` command. If your system doesn't have this command, it can be installed using the package manager of your distribution.
+
+   For Ubuntu:
+
+   ```sh
+   sudo apt update -y
+   sudo apt install procps
+   ```
+
+   Run this command to determine the PID of the Jellyfin server:
+
+   ```sh
+   ps aux
+   ```
+
+   The above command will display a list of processes similar to the following:
+
+   ```cmd
+   # ps aux
+   USER       PID %CPU %MEM    VSZ   RSS TTY      STAT START   TIME COMMAND
+   root         1  1.8  7.4 276905428 2304952 ?   Ssl  Sep04  17:23 /jellyfin/jellyfin
+   root       914  0.0  0.0   2480   580 pts/0    Ss   05:33   0:00 sh
+   root      2171  0.0  0.0   6756  2940 pts/0    R+   12:55   0:00 ps aux
+   ```
```

Comment:
maybe just `ps aux | grep /jellyfin` ?

Reply:
that would not work for any process running actually barebones. I dont have a barebones install so /jellyfin/jellyfin is actually the path docker uses.

Reply:
Why not? Isn't `ps aux | grep /jellyfin` just feeding the output of `ps aux` to find lines that contain `/jellyfin`? In the docs you suggested that the right most part of the command will always be `/jellyfin`